source/docs/appendices/migration-guides.md

```diff
@@ -26,6 +26,53 @@ The properties that were tagged as deprecated in `theme/components/atoms/Button/
 
 Please update your code to fit the new behavior.
 
+### Updated the Prismic Module
+
+#### Enforce trailing slash
+
+To enforce a trailing slashes in your url, you can simply add a trailing slash to the `path` on the `registerRouteableType` method. This will redirect a non-trailing url to the trailing url to ensure consistency. The same behavior is also applied for paths without a trailing slash. here is an example:
```

Comment:
```suggestion
To enforce a trailing slash in your url, you can simply add a trailing slash to the `path` on the `registerRouteableType` method. This will redirect a non-trailing url to the trailing url to ensure consistency. The same behavior is also applied for paths without a trailing slash. here is an example:
```